### ISSUE TITLE
Upload slow

### ISSUE THREAD

**Issue by Aburazak** [AdguardTeam/AdGuardDNS#747]
### Issue Details

Very slow upload speed when using ADGRUAD DNS IP

**Comment by BabyClasher23** [comment]
Yes, upload speeds on sites like Google Drive & YouTube are painfully slow while using AdGuard DNS. I've tried with multiple ISP's and the problem is still there. Please guys do something about it

**Comment by GulnazSalakhova** [comment]
Hello, sorry for the long answer. Could you please confirm whether the issue still persists?

**Comment by BabyClasher23** [comment]
Yes, the issue still persists.

**Comment by GulnazSalakhova** [comment]
Could you please provide some details:

- Exact DNS address/protocol used - which AdGuard DNS server (e.g., Default, Family, Non-filtering) and protocol (plain DNS/53, DoH, DoT, DNSCrypt)?

- Side-by-side speed test: results with AdGuard DNS set vs. with your ISP's default DNS (or 8.8.8.8/1.1.1.1) on the same network, same time - both download and upload numbers.

- Is the slowdown specific to Google Drive/YouTube, or do other upload-heavy services (e.g., Dropbox, direct file upload tests) show the same drop?

- Traceroute/MTR output to the AdGuard DNS server IP you're using, taken while the slowdown occurs.

- Are you using AdGuard DNS only as a resolver, or also through an AdGuard app/VPN/proxy at the same time?

- Result of the AdGuard DNS test page (https://adguard.com/en/test.html) - please share which server/location it shows you're connected to.

- This will help confirm whether the bottleneck is actually DNS resolution or something else in the routing path.

**Comment by GulnazSalakhova** [comment]
Closed due not reply
